src/apis/avm/inputs.ts

```diff
@@ -22,7 +22,7 @@ const serializer = Serialization.getInstance();
  * @returns An instance of an [[Input]]-extended class.
  */
 export const SelectInputClass = (inputid:number, ...args:Array<any>):Input => {
-  if (inputid === AVMConstants.SECPINPUTID) {
+  if (inputid === AVMConstants.SECPXFERINPUTID || inputid === AVMConstants.SECPXFERINPUTID_CODECONE) {
```

Comment:
Similar to my comment on `*_CODECONE`, this is probably premature, but I wonder if there's an easy way to say "any sexpxferinputid"

Reply:
Similar answer. We can brainstorm on a better architecture after we get apricot out the door. For now I want to ship this and spend some more time extensively QAing to make sure there are no obvious regressions.